src/Ganeti/HTools/Loader.hs

```diff
@@ -388,11 +388,10 @@ setStaticKvmNodeMem nl static_node_mem =
 -- | Update node memory stat based on instance list.
 updateMemStat :: Node.Node -> Instance.List -> Node.Node
 updateMemStat node il =
-    let missing = truncate (Node.tMem node)
-                  - Node.nMem node
-                  - Node.fMem node
-                  - nodeImem node il
-    in node { Node.xMem = missing }
+  let node2 = node { Node.iMem = nodeImem node il }
```

Comment:
Do we have lens support for these objects? It'd be so much cleaner to use that instead.